lib/audit/catalog_to_moab.rb

```diff
@@ -142,4 +143,23 @@ def online_moab_found?(druid, storage_dir)
     false
   end
 
+  # given whether the caller found the expected version of preserved_copy on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly.
+  def set_status_as_seen_on_disk(found_expected_version)
+    unless moab_validation_errors.empty?
```

Comment:
per tony's comment below, what if we did something like

```ruby
if moab_validation_errors.any?
  update_status(...)
  return
end
```

Do we prefer this?

If we like this, it seems to occur only in the analogous place in POHandler, so add a commit here changing that one too?  I would find it easier than "unless" statements ...

Reply:
i prefer it as is, but i was not initially a fan of `unless`, so i get the aversion.  my preference here is because in both the usages that get changed, there's a neighboring `unless` statement.  having both be `unless` feels more immediately readable to me.

but i don't feel that strongly about it, so i pushed a commit with the suggested change.  i'd prefer to remove it before merging, but if everyone prefers this style, feel free to merge (assuming no new complaints otherwise).